docs/tutorials/tabular_prediction/tabular-quickstart.md

```diff
@@ -86,7 +86,7 @@ From this summary, we can see that AutoGluon trained many different types of mod
 
 ```{.python .input}
 print("AutoGluon infers problem type is: ", predictor.problem_type)
-print("AutoGluon categorized the features as: ", predictor.feature_types)
+print("AutoGluon categorized the features as: ", predictor.feature_metadata.type_map_raw)
```

Comment:
It'd be nice if user could print the feature_metadata without having to remember such a long name

Reply:
do you have a suggestion to make it shorter? I wouldn't want predictor to have a type_map_raw object itself as that is overly limiting. We could do `print(predictor.feature_metadata)`, but this would have to be specialized to give all the information (raw + special).

Reply:
I just think it'd be good to have one "official" way of viewing this info with 1 simple line of code (that prints out all attributes of the `feature_metadata` object including both raw & special features; maybe the latter only if there are any special features to print though). 

Either `print(predictor.feature_metadata)` or something like `predictor.feature_info()` seem good to me.

Reply:
Lets clarify in future PR. For now, print(predictor.feature_metadata) should do the trick.